fastqc/0.12.1/Dockerfile

```diff
@@ -0,0 +1,90 @@
+ARG FASTQC_VER="0.12.1"
+
+# FROM defines the base docker image. 
+# The 'as' keyword lets you name the following stage. We use `app` for the production image
+
+FROM ubuntu:focal as app
+
+#re-instantiate these variables
+ARG FASTQC_VER
+
+# metadata
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="FASTQC"
+LABEL software.version="${FASTQC_VER}"
+LABEL description="A quality control analysis tool for high throughput sequencing data"
+LABEL website="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
+LABEL website.code.repo="https://github.com/s-andrews/FastQC/"
+LABEL license="https://github.com/s-andrews/FastQC/blob/master/LICENSE.txt"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+LABEL maintainer3="Pooja Gupta"
+LABEL maintainer3.email="biopooja@gmail.com"
+
+# install dependencies; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    unzip \
+    wget \
+    perl \
+    default-jre \
+    ca-certificates \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+# install fastqc. Make /data for use as a working dir
+RUN wget https://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v${FASTQC_VER}.zip && \
+    unzip fastqc_v${FASTQC_VER}.zip && \
+    rm fastqc_v${FASTQC_VER}.zip && \
+    chmod +x FastQC/fastqc && \
+    mkdir /data
+
+# set PATH and working directory
+ENV PATH="${PATH}:/FastQC/"
+WORKDIR /data
+
+# TEST STAGE - Getting Test Data #
+# getting fastq files with SRA tools - input data
+FROM ncbi/sra-tools:3.0.1 as fastq
+
+# set working directory to /fastq_test
+WORKDIR /fastq_test
+
+RUN fasterq-dump --threads 4 -A SRR6082043 && \
+    gzip *.fastq && \
+    mv SRR6082043_1.fastq.gz test_1.fastq.gz && mv SRR6082043_2.fastq.gz test_2.fastq.gz
+
+# new base for testing
+FROM app as test
+
+#re-instantiate these variables
+ARG FASTQC_VER
```

Comment:
I couldn't see where the FASTQC_VER argument was used in the test layer. 

Am I missing something?

Reply:
You're correct, it's not being used. Doesn't hurt to keep it but I can delete it from the dockerfile if you'd like